tmt/plans/edge-test.fmf

```diff
@@ -0,0 +1,75 @@
+summary: Edge commit test
+discover:
+  how: fmf
+  test: edge-test
+execute:
+  how: tmt
+provision:
```

Comment:
You should add required cpu and memory here.

Reply:
fixed